types/fparser/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for fparser 2.0
+// Project: https://github.com/bylexus/fparse
+// Definitions by: bylexus <https://github.com/bylexus>,
+//                 andrew0687 <https://github.com/andrew0687>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="./fparser.d.ts"/>
+
+export default FormulaC;
```

Comment:
```suggestion
import Formula from './src/fparser';
export = Formula;

export as namespace Formula;
```
you can find this pattern in UMD lib documentation, should cover all usage of `fparser`.